include/deal.II/hp/fe_collection.h

```diff
@@ -188,6 +188,27 @@ namespace hp
     bool hp_constraints_are_implemented () const;
 
     /**
+     * Try to find a least dominant finite element inside this FECollection
+     * which dominates other finite elements provided as fe_indices in @p fes .
+     * For example, if FECollection consists of {Q1,Q2,Q3,Q4} and we are looking
+     * for the least dominant FE for Q3 and Q4 (@p fes is {2,3}), then the
+     * answer is Q3 and therefore this function will return its index in
+     * FECollection, namely 2.
+     *
+     * If we were not able to find a finite element, the function returns
+     * numbers::invalid_unsigned_int .
+     *
+     * Note that for the cases like when FECollection consists of
+     * {FE_Nothing(),Q1} with @p fes = {1}, the function will not find the most
+     * dominating element as the default behavior of FE_Nothing is to return
+     * FiniteElementDomination::no_requirements when comparing for face domination.
+     * This, therefore, can't be considered as a dominating element in the sense
+     * defined in FiniteElementDomination .
```

Comment:
Aw, that's awkward. I would have guessed that if the input set has only one element, that that finite element canonically wins the competition. After all, it dominates all in the set, and among all dominating elements, it is (at least one of) the weakest for sure.